From: David Holsgrove <david.holsgrove@xilinx.com>
Date: Mon, 16 Dec 2013 16:37:32 +1000
Subject: microblaze: Add build_gdbserver=yes to top level configure.tgt

For Microblaze linux toolchains, set the build_gdbserver=yes
to allow driving gdbserver configuration from the upper level

This patch has been absorbed into the original patch to add
linux gdbserver support for Microblaze.

Signed-off-by: David Holsgrove <david.holsgrove@xilinx.com>
Upstream-Status: Pending
---
 gdb/configure.tgt |    1 +
 1 file changed, 1 insertion(+)

diff --git a/gdb/configure.tgt b/gdb/configure.tgt
index 1285aac..6ea10a2 100644
--- a/gdb/configure.tgt
+++ b/gdb/configure.tgt
@@ -342,6 +342,7 @@ microblaze*-linux-*|microblaze*-*-linux*)
 	gdb_target_obs="microblaze-tdep.o microblaze-linux-tdep.o microblaze-rom.o glibc-tdep.o \
 			monitor.o dsrec.o solib-svr4.o symfile-mem.o linux-tdep.o"
 	gdb_sim=../sim/microblaze/libsim.a
+	build_gdbserver=yes
 	;;
 microblaze*-*-*)
 	# Target: Xilinx MicroBlaze running standalone
-- 
1.7.9.5

